app/views/locations/_show_machines.html.haml

```diff
@@ -1,9 +1,8 @@
-= banner('show_machines_location', 'SHOW MACHINES', location)
-%div[location, :show_machines]{:style => 'display:none'}
+%div[location, :show_machines]
 
 :javascript
   $(function() {
```

Comment:
I think I can remove this entire function. There's no banner anymore, and this function is called from the js file. I'll work on it tonight.